composer.json

```diff
@@ -13,16 +13,17 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.3|^8.0",
         "psr/http-factory": "^1.0",
         "psr/http-message": "^1.0",
         "psr/http-server-handler": "^1.0",
         "psr/http-server-middleware": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5",
+        "phpunit/phpunit": "^9.0",
         "phpspec/prophecy": "^1.10",
-        "squizlabs/php_codesniffer": "^3.5.8"
+        "squizlabs/php_codesniffer": "^3.5.8",
+        "phpspec/prophecy-phpunit": "^2.0"
```

Comment:
nitpick: can we alphabetically order the deps

Reply:
Sure!

Added `sort_packages` config too for the future too 👍